kubeflow/mnist/k8s/pytorch_job_mnist.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: "kubeflow.org/v1"
+kind: "PyTorchJob"
+metadata:
+  name: "pytorch-dist-mnist-nccl"
+spec:
+  pytorchReplicaSpecs:
+    Master:
+      replicas: 1
+      restartPolicy: OnFailure
+      template:
+        metadata:
+          annotations:
+            sidecar.istio.io/inject: "false"
+        spec:
+          containers:
+            - name: pytorch
+              image: navarrepratt/pytorch_dist_mnist:1
+              args:
+                - "--backend"
+                - "nccl"
+                - "--log-dir"
+                - "/mnt/pvc/pytorch/logs"
+                - "--data-dir"
+                - "/mnt/pvc/dataset"
+                - "--model-dir"
+                - "/mnt/pvc/pytorch/checkpoints"
+              resources:
```

Comment:
Do launchers in PyTorch actually do compute? Ie it needs a GPU? Otherwise. Remove the GPU request/limit